howso/client/base.py

```diff
@@ -2540,28 +2509,7 @@ def react(  # noqa: C901
                 suppress_warning=suppress_warning
             )
 
-        # Convert new detail keys that were returned back to the requested ones
-        if detail_response := response.get('details'):
-            for key in deprecated_keys_used:
-                new_key = _RENAMED_DETAIL_KEYS[key]
-                if new_key in detail_response:
-                    detail_response[key] = detail_response[new_key]
-                    del detail_response[new_key]
-                if key in _RENAMED_DETAIL_KEYS_EXTRA.keys():
-                    # This key has multiple return keys that should be renamed to the old:
-                    for extra_old_key, extra_new_key in _RENAMED_DETAIL_KEYS_EXTRA[key]['additional_keys'].items():
-                        if extra_new_key in detail_response:
-                            detail_response[extra_old_key] = detail_response[extra_new_key]
-                            del detail_response[extra_new_key]
-
-            if "categorical_action_probabilities" in detail_response:
-                detail_response["categorical_action_probabilities"] = internals.update_caps_maps(
-                    detail_response["categorical_action_probabilities"],
-                    feature_attributes
-                )
-
-        return Reaction(response.get('action'), detail_response)
-
+        return Reaction(response.get("action"), response.get("details"), feature_attributes)
```

Comment:
Maybe access these by key instead to correct some type warnings here and in the other reacts. They should always exist from the response from format_react_response